### PR TITLE
Update the copy of the Experiments page

### PR DESCRIPTION
**Pull request by Copons**
## Description

* Update the copy and fix the capitalization of the Experiments page.
* Also adds a note to the Full Site Editing experiment, warning that it's still a WIP feature and could potentially break the site.

## Testing instructions

Open `/wp-admin/admin.php?page=gutenberg-experiments` and make sure it looks fine.

### CODE REVIEW

File: lib/experiments-page.php [hunk 1]
```diff
@@ -80,7 +80,7 @@ function gutenberg_initialize_experiments_settings() {
 		'gutenberg-experiments',
 		'gutenberg_experiments_section',
 		array(
-			'label' => __( 'Enable Full Site Editing', 'gutenberg' ),
+			'label' => __( 'Enable Full Site Editing (Warning: this will break the site)', 'gutenberg' ),
```
[the remark this answers is not in the document]
Comment (Addison-Stavlo) [reply lib/experiments-page.php]: Seems like the warning is now hard to miss.  Looks like a good change.

> and then clarify below what it actually means. 

Perhaps it is worth considering clarifying below what "break the site" means, but I think the warning gets the main point across.

**Review by Addison-Stavlo — APPROVED** [review]
Il approve for what it's worth.  It definitely seems like a good fix to make that warning a bit more apparent while FSE is being developed.  It does look like we still have reviews requested from others though.

**Review by vindl — APPROVED** [review]

**Comment by talldan** [comment]
I see the need for an additional warning, but the current text does seem extreme. I agree that it might be worth elaborating some more on what happens (e.g. 'Warning: this will replace your theme and cause potentially irreversible changes to your site. Please consider using this only in a development environment.')

> This has been happening rather frequently on WordPress.com, which threw off track our customer support people that weren't aware of the Full Site Editing experiment.

Maybe WordPress.com needs some more control over whether the experiments are visible or be able to override specific experiments to be disabled using a filter or something similar.

**Comment by Copons** [comment]
@talldan Yup, the copy here was intended as a placeholder to get the discussion going.
I promised to involve some copy review once we decided this was the way to go.
I'll update this with your suggestion and then add the Needs Copy Review label. 👍

**Comment by michelleweber** [comment]
I like @talldan's wording, and have minor suggestions for that whole area -- some capitalization fixes, some rephrasing.

> **Experimental settings**
> 
> The block editor includes experimental features that are useable while they're in development. Select the ones you'd like to enable. These features are likely to change, so avoid using them in production.
> 
> - Enable Widgets screen and Legacy Widgets block
> - Enable Navigation Menu block
> - Enable block directory search
> - Enable Full Site Editing (Note: Warning: this will replace your theme and cause potentially irreversible changes to your site. Please consider using this only in a development environment.)

**Comment by Copons** [comment]
@michelleweber Thanks for the suggestions!
Though it seems you've based them upon the old screenshots I provided in the PR description.

Currently, the Experiment page is like this:

Screenshot 2019-12-13 at 14 41 13

It's not completely different, but a couple of experiments changed.

I can incorporate your suggestions in this PR, so I'm just double checking in case you wish to improve the copy of the new experiments too. 🙂

**Comment by michelleweber** [comment]
Ah, gotcha! Yes, there are some small changes, just to capitalization, that we should make in the current copy:

- Enable Widgets screen and Legacy Widgets block
- Enable block directory search
- Enable Full Site Editing (Note: Warning: this will replace your theme and cause potentially irreversible changes to your site. We recommend using this only in a development environment.)
- Enable Full Site Editing demo templates
- Enable page templates

**Comment by Copons** [comment]
Thanks @michelleweber! I've updated the strings as per your suggestions and removed the Needs Copy Review label. 🙇

**Review by talldan — APPROVED** [review]
This looks good, thanks @Copons and @michelleweber!

**Comment by carlomanf** [comment]
Hi, don't know if this is the right place to ask, but can someone clarify what "potentially irreversible changes" these are referring to?

From what I can tell, everything is reverted back when the FSE is turned off?

**Pull request merged**